### ISSUE TITLE
(Master Regression) RDAirplay does not update CURRENT_LOG

### ISSUE THREAD

**Issue by deltecent** [ElvishArtisan/rivendell#317]
The CURRENT_LOG is no longer being saved in the LOG_MACHINES table.

**Comment by deltecent** [comment]
Any reason why this is commented out?
```
/*
void RDLogPlay::setLogName(QString name)
{
  if(logName()!=name) {
    RDLogEvent::setLogName(name);
    emit renamed();
    rda->airplayConf()->setCurrentLog(play_id,name.left(name.length()-4));
  }
}
*/
```

**Comment by deltecent** [comment]
Fixed #322
